### PR TITLE
[runtime] Support ONNX as an build option

### PR DESCRIPTION
**Pull request by pengzhendong**

### CODE REVIEW

File: runtime/core/patch/openfst/extensions/special/CMakeLists.txt [hunk 1]
```diff
@@ -0,0 +1,67 @@
+file(GLOB HEADER_FILES ../../include/fst/extensions/special/*.h)
```
Comment (robin1001) [inline runtime/core/patch/openfst/extensions/special/CMakeLists.txt:1]: what is the function of this file?

Comment (pengzhendong) [reply runtime/core/patch/openfst/extensions/special/CMakeLists.txt:1]: I just copy the file from this pr: https://github.com/kkm000/openfst/pull/36

Comment (robin1001) [reply runtime/core/patch/openfst/extensions/special/CMakeLists.txt:1]: do we use it now?

Comment (pengzhendong) [reply runtime/core/patch/openfst/extensions/special/CMakeLists.txt:1]: https://github.com/kkm000/openfst/pull/36/files#diff-d34b9d7d7cb7d326f283b20078ce57bcb89805249609fd257eccd3f069550b8bR38-R49

Please see the patch.

**Review by robin1001 — APPROVED** [review]

**Pull request merged**